Wallet template tutorial (#738)
scale-it/algo-builder: docs/tutorials/t-05.md

### Requirements

- [Vue CLI](https://cli.vuejs.org/guide/installation.html)<br/>
- [NodeJS](https://nodejs.org/en/download/) 14.11.0 or above<br/>
- JavaScript package manager - either [YARN](https://classic.yarnpkg.com/en/docs/install) version 3.1.0 or above or NPM which comes with NodeJS.<br/>
- IDE, e.g. [VSCode](https://code.visualstudio.com/download)<br/>
- Other dependencies are listed in [package.json](https://github.com/scale-it/algo-builder-templates/blob/master/wallet/package.json). Install them with yarn install or npm install.<br/>

### Background
This is a Web Application created using Vue through which one can connect to Algorand Blockchain and perform transaction (for eg: send Algo or call application) by connecting to your algorand wallet, select different accounts available on different networks such as MainNet, TestNet, BetaNet.

### Steps

>  #### 1. Development
>    The source code for this tutorial can be found on GitHub. Start by cloning the repository:<br/>
>    `git clone https://github.com/scale-it/algo-builder-templates.git`
> 
>    Change into the project directory:<br/>
>    `cd algo-builder-templates`

Reviewer: ```suggestion
#### 1. Setup
The source code for this tutorial can be found on GitHub. Start by cloning the repository

    git clone https://github.com/scale-it/algo-builder-templates.git
    cd algo-builder-templates/wallet
```

>    The repository contains the following templates:<br/>

Reviewer: why there is this additional indentation?

   **htlc** : This is a descriptive example explaining how HTLC contracts work with Algo-Builder.<br/>
   **monitor-payments**: This is a nodejs app which monitors payment transactions to a particular address.<br/>
   **shop**: This demonstrates the usage of a payment widget to trigger a purchase by connecting to different wallets.<br/>
   **wallet**: This helps in connecting to different wallets.<br/>
   **warcraft**: This is a descriptive example explaining how Escrow contract work with Algo-Builder.<br/>

>    Migrate to wallet directory:<br/>
>    `cd wallet`
> 

Reviewer: ```suggestion
```

 #### 2. Environment Setup
   Install all dependencies in your environment:<br/>
   ` yarn install` OR `npm install`

   Now, run the app for local development. It will automatically update when there are any changes.<br/>

>    `yarn run dev`

Reviewer 2: ```suggestion
   `yarn run dev` or `npm run dev`
```

Author: oh yes, thanks for reminding

   After successfully running, your browser screen will look similar to this.

>    <img src="./t-05/assets/homescreen.png" height="200" title="Home Page" />

Reviewer: please rename this file to `t-wallet.md` and create `t-wallet` directory for additional assets

Author: updated the names

> 

Reviewer: ```suggestion

#### 3. Using the Wallet Widget
```

   Now you can select your preferred network and connect to the wallet as per your choice.
   <img src="./t-05/assets/address_selected.png" height="200" title="Account Selected" />

   You can now interact with the algorand blockchain, by sending algos, or call an application.
   <img src="./t-05/assets/algo_transaction.png" height="200" title="Send Transaction" />

   You can also update the address to which algos are sent, the algo amount or the application ID by configuring it in `src/constants`.

>    ```js

Reviewer 2: ```suggestion
   ```ts
```

   export const toAddress = "WK73JQQGCRQCZ5GLQHCXXB6OXIGF6SD22F5P7HPXF5PNH23YUUALUMKOZ4";
   export const amount: number = 10e6;
   export const applicationId: number = 189;
   ```

>    And finally you can disconnect your chosen wallet.

Reviewer: Add one more step explaining how user can add style, and add a screenshot with a link to an example (the SigmaWallet) showing how the wallet can look like with proper styling (only take a screenshot of the wallet widget)

Author: @robert-zaremba  added the new step, Could you take a look at it.?
